client/src/functions/SelectRandomFromList.js

```diff
@@ -0,0 +1,10 @@
+export default function SelectRandomFromList(inputArray, amount) {
```

Comment:
Praise: I'm liking these changes, really clever stuff! I'm glad you added the amount variable, that solves the issue I was going to ask a question about 😆 
So the loop will keep running as long the number of songs in randomList is less than the amount i.e. 5 songs. Awesome.